src/rgw/rgw_aws_sns.h

```diff
@@ -0,0 +1,25 @@
+
+//
+// Created by prateek on 29/03/20.
+//
+
+#pragma once
+
+#include <string>
+#include "include/common_fwd.h"
+#include <aws/sns/SNSClient.h>
```

Comment:
please use forward declaration instead of "include"
(reduce build dependencies)